docs/how-to/communicate.example.c

```diff
@@ -32,7 +26,7 @@ amongoc_box after_hello(amongoc_box state_ptr, amongoc_status*, amongoc_box resp
     bson_view resp = bson_view_from(amongoc_box_cast(bson_doc, resp_data));
     // Just print the response message
     fprintf(stdout, "Got response: ");
-    print_bson(stdout, resp, mlib_str_view_from(""));
+    bson_write_repr(stderr, resp);
```

Comment:
```suggestion
    bson_write_repr(stdout, resp);
```

Suggest writing to stdout since no error occurred.